### PR TITLE
Update non functional requirements

### PR DESCRIPTION
**Pull request by matthewyip1511**
As discussed, I have updated the NFRs as I have noted that this has not been worked on in PR #30. Thanks!

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -339,8 +339,13 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 1.  Should work on any _mainstream OS_ as long as it has Java `17` or above installed.
 2.  The data should be stored locally in a JSON file.
-3.  Should be able to hold up to 1000 persons without a noticeable sluggishness in performance for typical usage.
-4.  A user with above average typing speed for regular English text (i.e. not code, not system admin commands) should be able to accomplish most of the tasks faster using commands than using the mouse.
+3.  The data should be stored in a way that complies with the Personal Data Protection Act, Singapore [[PDPA](https://www.pdpc.gov.sg/overview-of-pdpa/the-legislation/personal-data-protection-act)]
+4.  Should be able to hold up to 1000 persons without a noticeable sluggishness in performance for typical usage.
+5.  A user with above average typing speed for regular English text (i.e. not code, not system admin commands) should be able to accomplish most of the tasks faster using commands than using the mouse.
+6.  The app should display results within 2 seconds when searching for a client.
+7.  The app should be live 24 hours a day.
```
[the remark this answers is not in the document]
Comment (zi-yii) [reply docs/DeveloperGuide.md:346]: Good catch

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -339,8 +339,13 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
+8.  The app should save the data entered even if it exits unexpectedly while running.
+9.  The app should have a log of user activity to identify malicious actors
```
[the remark this answers is not in the document]
Comment (yanqiyqh) [reply docs/DeveloperGuide.md:348]: I agree, good catch!

**Review by zi-yii — APPROVED** [review]
Very good 😆

**Review by RezwanAhmed123 — APPROVED** [review]
LGTM

**Pull request merged**